src/__tests__/balanceCheck.test.ts

```diff
@@ -342,16 +342,16 @@ describe("transferToSummary and check balances", () => {
       {
         token_type: "native",
         tokenAddress: null,
-        amount: new BigNumber(0.33),
+        value: new BigNumber(0.33),
         receiver: testData.addresses.receiver1,
         decimals: 18,
         symbol: "ETH",
         receiverEnsName: null,
       },
     ];
     const summary = assetTransfersToSummary(transfers);
-    expect(summary.get(testData.unlistedERC20Token.address)?.amount.toFixed()).to.equal("6.4");
-    expect(summary.get(null)?.amount.toFixed()).to.equal("3.33");
+    expect(summary.get(testData.unlistedERC20Token.address)?.value.toFixed()).to.equal("6.4");
+    expect(summary.get(null)?.value.toFixed()).to.equal("3.33");
```

Comment:
Should we also have a test that works for amount as well? I don't think its absolutely necessary because there will be a lot of duplicated code.